dev/TabView/TabView.xaml

```diff
@@ -391,6 +393,15 @@
                                     <VisualTransition GeneratedDuration="0:0:0.2" To="NotDragging" />
                                 </VisualStateGroup.Transitions>
                             </VisualStateGroup>
+
+                            <VisualStateGroup x:Name="IconStates">
+                                <VisualState x:Name="Icon"/>
+                                <VisualState x:Name="NoIcon">
+                                    <VisualState.Setters>
+                                        <Setter Target="IconBox.Visibility" Value="Collapsed"/>
```

Comment:
"IconBox" - typo?  Should be IconControl?

Reply:
That's intentional. IconBox is the ViewBox around the IconControl, which has margin and such on it, so I want it to disappear also.

Reply:
Ahhh, gotcha.  I was doing a Ctrl+F for IconBox which was coming up empty, but it turns out it was oooone line away from being visible in the CR sections.  I see now.